holoviews/selection.py

```diff
@@ -645,7 +649,9 @@ def __init__(self, color_prop='color', alpha_prop='alpha', backend=None):
         self.alpha_props = [alpha_prop]
         self.backend = backend
 
-    def build_selection(self, selection_streams, hvobj, operations, region_stream=None, cache={}):
+    def build_selection(self, selection_streams, hvobj, operations, region_stream=None, cache=None):
```

Comment:
Is it okay to change cache here?

Reply:
Yes, I think that's correct. The idea is that you can pass in an external cache.